kubemarine/resources/configurations/globals.yaml

```diff
@@ -48,7 +48,7 @@ nodes:
       timeout: 10
       retries: 30
   boot:
-    reboot_command: 'reboot 2>/dev/null >/dev/null &'
+    reboot_command: 'systemctl stop sshd && sudo reboot 2>/dev/null >/dev/null'
```

Comment:
Sometimes reboot is stuck. The problem is relatively stably reproduced on some deployer environments, but is still floating.

Just after reboot in [system.perform_group_reboot](https://github.com/Netcracker/KubeMarine/blob/0.18.2/kubemarine/system.py#L377) we [disconnect](https://github.com/Netcracker/KubeMarine/blob/0.18.2/kubemarine/core/group.py#L687) fabric Connections and then try to run `last reboot`. Running of commands internally creates paramiko client via [Connection.open()](https://github.com/fabric/fabric/blob/2.6.0/fabric/connection.py#L636) and then `Connection.run` opens paramiko channel.

For unknown reason in some cases, paramiko client is opened successfully just after reboot (probably because reboot procedure did not kill sshd daemon yet), but [opening of channel](https://github.com/paramiko/paramiko/blob/2.11.1/paramiko/transport.py#L1045-L1055) is stuck for 1 hour.

```
Thread 0x00007f88237fe700 (most recent call first):
  File "/usr/local/lib/python3.11/threading.py", line 324 in wait
  File "/usr/local/lib/python3.11/threading.py", line 622 in wait
  File "/usr/local/lib/python3.11/site-packages/paramiko/transport.py", line 1046 in open_channel
  File "/usr/local/lib/python3.11/site-packages/paramiko/transport.py", line 920 in open_session
  File "/usr/local/lib/python3.11/site-packages/fabric/connection.py", line 700 in create_session
  File "/usr/local/lib/python3.11/site-packages/fabric/connection.py", line 30 in opens
  File "<decorator-gen-2>", line 2 in create_session
  File "/usr/local/lib/python3.11/site-packages/fabric/runners.py", line 36 in start
  File "/usr/local/lib/python3.11/site-packages/invoke/runners.py", line 431 in _run_body
  File "/usr/local/lib/python3.11/site-packages/invoke/runners.py", line 380 in run
  File "/usr/local/lib/python3.11/site-packages/invoke/context.py", line 102 in _run
  File "/usr/local/lib/python3.11/site-packages/fabric/connection.py", line 723 in run
  File "/usr/local/lib/python3.11/site-packages/fabric/connection.py", line 30 in opens
  File "<decorator-gen-3>", line 2 in run
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58 in run
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 83 in _worker
  File "/usr/local/lib/python3.11/threading.py", line 975 in run
  File "/usr/local/lib/python3.11/threading.py", line 1038 in _bootstrap_inner
  File "/usr/local/lib/python3.11/threading.py", line 995 in _bootstrap
```

I suggest to stop the sshd daemon just before the reboot. In practice, Connection.open() was never successful in many attempts, but it uses the connection timeout of 10 seconds that we pass. It is thus finished after a number of attempts.

Also removed `&` because running of `reboot` in background does not work. https://www.fabfile.org/faq.html#why-can-t-i-run-programs-in-the-background-with-it-makes-fabric-hang

This has the drawback that if sshd dameon is stopped but reboot does not happen, connection via ssh will be lost.